environment.py

```diff
@@ -142,6 +142,11 @@ def env() -> Mapping[str, Optional[str]]:
         #
         'azul_current_catalog': None,
 
+        # A space-separated list of the sources in the current catalog to
+        # operate on, or "*" for all sources in the catalog.
+        #
+        'azul_current_sources': None,
```

Comment:
```suggestion
        'azul_current_sources': '*',
```

Reply:
Note that this causes warnings to be emitted. See my other comment.

Reply:
I'm OK for now with the false warnings. We get these in at least one other situation when we want to legitimately override an environment variable on the command line. I've opened 

https://github.com/DataBiosphere/azul/issues/6320

to track that.